proposals/interface-types/Explainer.md

```diff
@@ -346,7 +369,8 @@ used in the same adapter function:
     arg.get $str
     string-to-memory "mem" "malloc"
     call-export "frob_"
-    memory-to-string "mem" "free"
```

Comment:
We are still exporting our memory?

Reply:
As answered above: only from the core module, not the adapted module.  The core module is 100% encapsulated by the adapted module, so there is no issue, I think.  There's also no other way, that I know of, if we don't want to change core wasm semantics.